plugins/blip/basics/logic/actions.js

```diff
@@ -72,7 +105,7 @@ function clearSelected(opts) {
   return opts;
 }
 
-function setSelected(opts, selectedKey) {
+basicsActions.setSelected = function(opts, selectedKey) {
```

Comment:
This seemed like the DRY-est solution, given that I'd prefer to set the selected option before calling `setState()` [here](/tidepool-org/tideline/pull/299/files#diff-38db5d35ee21bffa99cd8dccaacb73ecR121).